src/clojure_by_example/ex05_immutability_and_fp.clj

```diff
@@ -421,26 +417,31 @@ habitable-planets ; confirm by checking the value of this
 
 ;; We can also alias the whole hash-map:
 
-(let [add-message (fn [{:keys [pname moons]
```

Comment:
These function bodies are starting to get bigger, and putting them in `let` blocks makes them harder to visually parse.

Reply:
Right, let's pull them out. I can do it... Should I?

Reply:
Oh you did it... I missed that. :100: